infra/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.25.0, <= 4.74, != 4.75.0"
+      version = ">= 4.80.0"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 3.25.0, <= 4.74, != 4.75.0"
```

Comment:
We can override the `<= 4.74` constraint because the issue that constraint was avoiding has been fixed in 4.75.1 (see [4.75.1 release notes](https://github.com/hashicorp/terraform-provider-google/releases/tag/v4.75.1)).